tests/integration/it_table_shapes_test.py

```diff
@@ -101,14 +102,9 @@ def test_edit(client):
     assert db_result.picture == update_data["picture"]
 
 
-@pytest.mark.skip(reason="create method doesn't have form.save()")
 def test_delete(client):
-    client.post(flask.url_for("table_shape.create"), data={
-        "description": "It's new shape",
-        "picture": "http://...."
-    })
-    identifier = models.TableShape.query.first().id
-
-    result = client.post(flask.url_for('table_shape.delete', id=identifier))
-    assert result.status_code == HTTPStatus.FOUND
-    assert models.TableShape.query.filter_by(id=identifier).count() == 0
+    table_shape = factories.TableShapeFactory()
+    response = client.post(
+        flask.url_for('table_shape.delete', id=table_shape.id))
```

Comment:
@vryazanov before deleting it, shouldn't you call ``/create`` to post it on the server? Otherwise, what is actually deleted?

Reply:
@amihaiemil  `factories.TableShapeFactory()` creates an instance in DB which we are going to delete